### DISCUSSION TITLE
Old GUI version

### DISCUSSION THREAD

**Discussion by R2rino** [devcompl/TTController#22] [Configuration] [answered]
Hello! Could you please provide a link with the old GUI version of your program? (v1.7) The one on the first page doesn't seems to work anymore. Thank you!

**Comment by MoshiMoshi0** [comment, marked as answer]
Seems to be working fine for me. 
But here: [RiingPlusController_v1.7.zip](https://github.com/MoshiMoshi0/TTController/files/3398830/RiingPlusController_v1.7.zip)
